clayui.com/content/docs/examples.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Examples'
+order: 5
+---
+
+import ClayLayout from '@clayui/layout';
+import ClayLink from '@clayui/link';
+import ClayList from '@clayui/list';
+
+## What is Storybook and how do we use it?
+
+Storybook is a user interface development environment and playground for UI components.
+We use it to create components independently and showcase components interactively in an isolated development environment.
+A Storybook is a collection of stories. Each story represents a single visual state of a component.
+
+## How you can leverage Storybook to make using Clay easier
+
+Whenever you have doubts about how a component is implemented or used, checking out its stories
+is a great way of getting valuable information about how we envisioned that component be used.
+
+You can use Storybook for inspiration, as well as guidelines, for your use case as well as getting insight into the composition of a component.
+
+## Examples/Demos
+
+Here are our demos we have in Storybook that utilize Clay components showcasing some common use cases.
+
+<ClayLayout.Container>
+	<ClayLayout.Row justify="start">
+		<ClayLayout.Col md={4} size={4}>
+			<ClayList flex>
+				<ClayLink
+					href="https://storybook.clayui.com/?path=/story/demos-templates--list-page"
+					target="_blank"
+				>
+					<ClayList.Item>
```

Comment:
`Item` needs to be the first descendant of `ClayList`, otherwise it won't be semantic.

Also, I think it'd be better to not use List and instead just ClayLink with line breaks between each item.  I think the list looks a little odd for this case.

Reply:
Makes me wonder whether there should be a check (only when `process.env.NODE_ENV === 'development'`) that looks at the children and makes sure there are no non-list children... (or at least no children that are definitely the wrong type of Clay component).

Reply:
This attempt of mine might have demonstrated a need for a new prop (maybe a boolean `link`) for `ListItem`. I wanted to make the whole List Item a link, not just the text inside. 

What're your thoughts on this prop @bryceosterhaus @matuzalemsteles ?

Reply:
@kresimir-coko I think we would still have problems with semantics. To reverse this problem we would need to modify `ClayList` and `ListItem` to not use `ul` and `li` for cases with `Link` or `Button`, I don't know if this would be very friendly.

Reply:
@matuzalemsteles not sure it's not bypassable tho, I think that with some CSS we can make the `a` full width and height and thus making the entire area of the `li` clickable.

Reply:
@kresimir-coko I'm not sure what you want to say, would it be changing the CSS to support this? I was referring that we could replace the tag `ul` and `li` with a `div` and `a` or `button` using the same classes, this is the most recommended for these cases, I suppose Clay already supports this since it extends from the bootstrap.

Reply:
@matuzalemsteles Yes, I'm suggesting that we keep the `ul` > `li` > `a` hierarchy, and just manipulate the css so that we can have the `a` element cover 100% of the `li`'s size

Reply:
Oh, now I get it 😁, Hmm 🤔. @pat270 what do you think?